src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -7,9 +7,10 @@ <h1>Home</h1>
   </div>
 </div>
 
+<!-- altered by Albert Yu  -->
```

Comment:
Please remove this. Your GitHub credentials will be reflected in the commit history if this PR is merged.